dm/templates/logsink/logsink.py

```diff
@@ -89,36 +120,32 @@ def create_storage(context, logsink_name):
         dest_prop['name'] = bucket_name
         dest_prop['project'] = project_id
         bindings = dest_prop.get('bindings', [])
-        bindings.append(
-            {
-                'role': 'roles/storage.admin',
-                'members': ['$(ref.' + logsink_name + '.writerIdentity)']
-            }
-        )
+        bindings.append({
+            'role': 'roles/storage.objectAdmin',
```

Comment:
This is breaking out bats test!

Reply:
in test file templates/logsink/tests/integration/logsink.bats, line 169)
#   `[[ "$output" =~ "roles/storage.admin" ]]' failed

https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/dm/templates/logsink/tests/integration/logsink.bats